mongo/mongo.go

```diff
@@ -17,8 +17,8 @@ const (
 
 // keep tags in sync with above const
 type Timestamps struct {
-	LastUpdated     time.Time           `bson:"last_updated,omitempty"     json:"last_updated,omitempty"`
-	UniqueTimestamp bson.MongoTimestamp `bson:"unique_timestamp,omitempty" json:"-"`
+	LastUpdated     time.Time            `bson:"last_updated,omitempty"     json:"last_updated,omitempty"`
```

Comment:
Should this also be a pointer or remove omitempty from bson and json tags?

Reply:
`time.Time` has a _zero value_ which is considered sufficient (and recommends using `mydoc.LastUpdated.IsZero()`)

from https://golang.org/pkg/time/#Time